assignments/closure.js

```diff
@@ -1,17 +1,25 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const myAwesomeClosure = (substring) => `Greeting from the Awesome Closure. ${substring}` ;
 
+const myAmazingFunction = (message) => myAwesomeClosure(message);
 
+console.log(myAwesomeClosure("How can I help you?"));
```

Comment:
this isnt quite right, closures have to do with scope, and thus require nested blocks of code.
should looks something like this:
```
function myAwesomeClosure(message) {

            function myAmazingFunction(){
                return `Greeting from the awesome closure ${message}`
            }
            myAmazingFunction();
        }
        Console.log(myAwesomeClosure());
```
as a side note just for now, avoid arrow functions when demonstrating closures, because they act slightly differently then regular syntax.

Reply:
Yup, closure is still on my relearn list